src/test/java/uk/gov/justice/probation/courtcaseservice/controller/model/CourtCaseRequestTest.java

```diff
@@ -51,8 +53,8 @@ void givenAllFieldsPresent_whenCallAsEntity_returnHearingEntity() {
                 .breach(BREACH)
                 .preSentenceActivity(PRE_SENTENCE_ACTIVITY)
                 .offences(Arrays.asList(
-                                new OffenceRequestResponse("OFFENCE_TITLE1", "OFFENCE_SUMMARY1", "ACT1", 10),
-                                new OffenceRequestResponse("OFFENCE_TITLE2", "OFFENCE_SUMMARY2", "ACT2", 20)
+                                new OffenceRequestResponse("OFFENCE_TITLE1", "OFFENCE_SUMMARY1", "ACT1", 10, Collections.emptyList()),
+                                new OffenceRequestResponse("OFFENCE_TITLE2", "OFFENCE_SUMMARY2", "ACT2", 20, Collections.emptyList())
```

Comment:
Updating the tests ATM.

Reply:
Currently we don't have a concrete use case for this data other than for emitting events, I think we can leave it for now and add it when we need it. If we're interested we can get it from the extended endpoint which is the main thing.